src/DayPicker.js

```diff
@@ -591,6 +591,10 @@ export class DayPicker extends Component {
     return (
       <div
         {...this.props.containerProps}
+        aria-modal="true"
+        aria-labelledby="IDREF"
```

Comment:
wouldn't this need to be an explicit id? Which component has this ID?

Reply:
That should have been changed to this https://github.com/emmadev/react-day-picker/pull/1/commits/b323fbc15b9e7073e64b3d07d7f2c66b5993c344

Reply:
👍